1428/en.md

```diff
@@ -0,0 +1,332 @@
+
+<h1 align="center"> <b>LightOj 1428 (Melody Comparison) </b> </h1>
```

Comment:
Add the title like this: https://github.com/lightoj-dev/problem-tutorials/pull/11/files#diff-49d89fcefb3733849902cae167206bfef3d2d2fa1569a2abcc5b303e53f828b9R1